features/src/cypress-deps/installer.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"builder/installer"
+	"fmt"
+	"os"
+)
+
+func main() {
+	if err := runMain(); err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func runMain() error {
+	fmt.Println("Installing Cypress Dependencies")
+	osInfo, err := installer.Tools.System.GetOsInfo()
+	if err != nil {
+		return fmt.Errorf("failed to get OS info: %w", err)
+	}
+
+	// Common dependencies for all supported distros
+	commonDeps := []string{
+		"libgbm-dev",
+		"libnotify-dev",
+		"libnss3",
+		"libxss1",
+		"libxtst6",
+		"xauth",
+		"xvfb",
+	}
+
+	var deps []string
+	if (osInfo.Vendor == "debian" && osInfo.Codename == "trixie") ||
```

Comment:
Don't you think this will probably also be valid for the next debian (14)? Maybe we should compare the version number and use `if version >= 13`?